lib/adagrams.rb

```diff
@@ -0,0 +1,153 @@
+require 'csv'
+
+def draw_letters
+  all_letters_hash = {
```

Comment:
Consider making this hash a global variable assigned outside the method so that it can be used other place.